JoyboyCommunity/src/modules/Tip/index.tsx

```diff
@@ -0,0 +1,112 @@
+import {Picker} from '@react-native-picker/picker';
+import {forwardRef, useMemo, useState} from 'react';
+import {Image, Platform, View} from 'react-native';
+import {Modalize as RNModalize} from 'react-native-modalize';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+import {LikeFillIcon} from '../../assets/icons';
+import {Button, Input, Modalize} from '../../components';
+import {Text} from '../../components/Text';
+import {useStyles} from '../../hooks';
+import {Spacing} from '../../styles';
+import stylesheet from './styles';
+
+export const TipToken = forwardRef<RNModalize>((props, ref) => {
+  const styles = useStyles(stylesheet);
+  const [token, setToken] = useState<string>('');
+  const [amount, setAmount] = useState<string>('');
+
+  const isActive = useMemo(() => {
+    if (amount.length > 0 && token.length > 0) {
+      return true;
+    }
+    return false;
+  }, [amount, token]);
+
+  return (
+    <Modalize
+      handlePosition={Platform.OS === 'ios' ? 'inside' : 'outside'}
+      adjustToContentHeight
+      title="Tip"
+      ref={ref}
+      modalStyle={styles.modal}
+    >
+      <SafeAreaView edges={['bottom', 'left', 'right']}>
+        <View style={styles.card}>
+          <View style={styles.cardHeader}>
+            <View style={styles.cardContent}>
+              <Image
+                style={{width: 48, height: 48}}
+                source={require('../../assets/joyboy-logo.png')}
+              />
```

Comment:
You can use Avatar component, it provides polygon effect, and you can pass the size as `<Avatar size={48} />` directly.